### ISSUE TITLE
Settings for High VRAM GPU

### ISSUE THREAD

**Issue by caniyabanci76** [tdrussell/diffusion-pipe#366]
Hi,

Are there any specific settings to utilize more the vram during training?  I'm currently trying out fine-tuning Qwen-Image on an A100 and noticing that VRAM usage is only around 37GB/80GB.  I'm guessing increasing the batch size and changing the dtype to fp16?

**Comment by jakehemmerle** [comment]
make sure gradient/activation checkpointing is disabled too. I think use bf16 for mixed precision
